addon/helpers/multi-promise-handler.js

```diff
@@ -0,0 +1,36 @@
+import Helper from '@ember/component/helper';
+import { hashSettled } from 'rsvp';
+/**
+ * This is the multi-promise-handler helper. It allows for passing in multiple network requests, while offering the
+ * ability to gracefully handle the response for each promise. The output would thus still leverage the suspense
+ * component data schema, but embedded to each response, there will be an assigned state.
+ * @example
+ *   {{#suspense-component
+ *     promise=(ember-async-component$multi-promise-handler promiseOne=promise.first promiseTwo=promise.second)
```

Comment:
Can we not use batmax syntax as an example since we don't bring that as a dependency. The consumers can decide how they want to consume this helper

Reply:
Addressed in latest commit